.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### General information
+
+  * Minishift version: 
+  * Download from: github releases / brew / chocolatey
+  * OS: Linux / macOS / Windows
+  * OS version/distro: 
+  * Hypervisor: KVM / xhyve / Hyper-V / VirtualBox
```

Comment:
If we are asking for output of `minishift start --shwo-libmachine-logs -v5` then we do not need to ask which hypervisor user is using. Also I am not sure if we get the OS information, Minishift Version etc in the output. If not then we should add these to the debug logs. basically we should ask very minimal things from user.